### ISSUE TITLE
`comma-dangle`: not detecting unnecessary last comma for arrow function arguments

### ISSUE THREAD

**Issue by Julien-Marcou** [eslint-stylistic/eslint-stylistic#1014]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

When using the `'@stylistic/comma-dangle': ['error', 'always-multiline']` rule, unnecessary commas are detected for normal function's parameters, but they are not detected for arrow function's parameters

```ts
// the unnecessary comma is detected
//                                     ↓
function addNumber(a: number, b: number,): number {
  return a + b;
}

// the unnecessary comma is NOT detected
//                                       ↓
const addNumberBis: (a: number, b: number,) => number = (a, b) => a + b;

```

### Reproduction

https://stackblitz.com/edit/github-fismmhdy

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
